assets/services.json

```diff
@@ -2016,6 +2031,7 @@
         "||ext-twitch.tv^",
         "||jtvnw.net^",
         "||ttvnw.net^",
+        "||twitch.amazon.com^",
```

Comment:
This domain redirects to gaming.amazon.com. I think it's better not to add it here.

Reply:
amazon should not be blocked here.